src/Frontend/util/__tests__/package-search-api.test.ts

```diff
@@ -111,4 +112,39 @@ describe('PackageSearchApi', () => {
       );
     });
   });
+
+  describe('getPackages', () => {
+    it('provides packages but no projects', async () => {
+      const packageInfo = faker.opossum.externalPackageInfo({
+        packageType: faker.packageSearch.system(),
+      });
+      const testPackage = faker.packageSearch.searchResponse({
```

Comment:
there is nothing gained here by prefixing variables with "test" or "mock".
```suggestion
      const testPackage = faker.packageSearch.searchResponse({
```